backend/dal/conversations.js

```diff
@@ -54,6 +54,7 @@ module.exports = (connection) => {
           .field('email')
           .field('viewed')
           .field('videoIsWatched')
+          .field('file_is_downloaded')
```

Comment:
Please create table 'events' with fields 'type' (not null), 'conversation'  (id, could be null) and 'metadata'
Ask any questions if you have